src/clr-angular/forms/common/label.ts

```diff
@@ -27,6 +27,10 @@ export class ClrLabel implements OnInit, OnDestroy {
   private subscriptions: Subscription[] = [];
   private enableGrid = true;
 
+  get labelText(): string {
+    return this.el.nativeElement.textContent;
```

Comment:
Need a platform check here.

Reply:
It was there, @shijir requested that it be removed in his feedback here: https://github.com/vmware/clarity/pull/3830#discussion_r327350898

It's easy to put back but would like agreement from everyone before I change it again. @shijir, any objections?

Reply:
We don't really need to inject another provider that wouldn't offer any value in this case. If we were accessing method beyond `this.el.nativeElement.textContent` like `this.el.nativeElement.textContent.doSomething()`, then sure let's check the platform. But we are only interested in `.textContent` property. If it's `undefined`, then let it be. It just won't be used. If it holds a text value, then we will use that. But what we could improve is this:
```
get labelText(): string {
    return this.el.nativeElement && this.el.nativeElement.textContent;
```

Reply:
I'm ok with what Shijir proposes, just don't want to have SSR problems.

Reply:
Done.